services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet | Movie:

```

Comment:
The return type hint should be `QuerySet` instead of `QuerySet | Movie` because the function returns a QuerySet of `Movie` instances.